index.js

```diff
@@ -217,7 +217,7 @@ function $asStringSmall (str) {
     }
   }
   if (last === 0) {
-    result = str
+    result += str
```

Comment:
can you try assigning last to `-1`  at the top of the function, and checking for `-1` in the if?

Reply:
It will not work, because we are using it inside the the loop: `result += str.slice(last, i) + '\\'`.
Slicing with a negative index as first value breaks some tests.